studio/schemas/documents/compensations.ts

```diff
@@ -3,6 +3,7 @@ import { titleSlug } from "../schemaTypes/slug";
 import seo from "../objects/seo";
 import { title } from "../fields/text";
 import { benefitId } from "./benefit";
+import MultiLineDescription from "../../components/MultiLineDescription";
```

Comment:
Not relevant to this PR. I also wouldn't want to add multilines to descriptions. It's a good idea but, but it also breaks with the default styling of Sanity, and Im not sure it's needed

Reply:
I agree it's not critical here, so we can remove it for now. I still think it's fair to expand on the Sanity design to allow multiple lines in a description though.